db/seeds.rb

```diff
@@ -9,9 +9,23 @@
 # bin/rails c    OR   rake db:reset AND bin/rails db:seed
 # rake db:drop db:create db:migrate db:seed
 
-6.times do |prize|
-  Prize.create(name: "Tshirt #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/t-shirt.svg", prize_type: 'Merch', description: 'This is a merch item', identifier: 'merch')
-  Prize.create!(name: "NFT #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+prizesList = [
+  {name: "T-shirt", percentage: 15, image: "assets/svgs/svgs/prizes/t-shirt.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "Pen", percentage: 15, image: "assets/svgs/prizes/pen.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "Hoodie", percentage: 15, image: "assets/svgs/prizes/hoodie.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "Keychain", percentage: 15, image: "assets/svgs/prizes/keychain.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "Tote Bag", percentage: 15, image: "assets/svgs/prizes/tote-bag.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "Hat", percentage: 15, image: "assets/svgs/prizes/hat.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch'},
+  {name: "CryptoPunks NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'},
+  {name: "Moonbirds NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'},
+  {name: "Pudgy Penguins NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'},
+  {name: "Cool Cat NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'},
+  {name: "Mutant Ape NFT", percentage: 1, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'},
+  {name: "Bored Ape NFT", percentage: 1, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT'}
+]
+
+12.times do |i|
+  Prize.create(prizesList[i])
 end
```

Comment:
Why not use `.map` on prizesList? 🤔

Reply:
Cool. Did you run the linter on this? I'm just wondering because of the `()`. I know both are valid in Ruby but haven't seen them used much.

Also not sure if this is the linter settings, but I guess this could be a one-liner? Added a `!` so you get some extra error info if it fails.

`prizes.map { |prize| Prize.create!(prize) }`

Reply:
`prizesList.map { |prize| Prize.create!(prize) }` works